stdlib/2/typing.pyi

```diff
@@ -62,46 +63,58 @@ _KT_co = TypeVar('_KT_co', covariant=True)  # Key type covariant containers.
 _VT_co = TypeVar('_VT_co', covariant=True)  # Value type covariant containers.
 _T_contra = TypeVar('_T_contra', contravariant=True)  # Ditto contravariant.
 
-class SupportsInt(metaclass=ABCMeta):
+def runtime(cls: _T) -> _T: ...
```

Comment:
Should the argument be a TypeVar bound to `Type[object]` or even `Type[Protocol]`, so that type checkers will give an error when this decorator is applied to something that's not a class?